python/build_all/linux/build.sh

```diff
@@ -31,8 +31,7 @@ process_args()
 process_args $*
 
 # instruct C builder to include python library and to skip tests
-
-./c/build_all/linux/build.sh --build-python $PYTHON_VERSION --skip-unittests $*
+./c/build_all/linux/build.sh --build-python $PYTHON_VERSION --skip-unittests --skip-samples --skip-service-client $*
```

Comment:
@damienpontifex - Can you please revert this line. We would like to build the samples & service client by default.